pyproject.toml

```diff
@@ -0,0 +1,15 @@
+[project]
+name = "aai-backend"
+version = "0.1.0"
+description = "Add your description here"
+readme = "README.md"
+requires-python = ">=3.13"
+dependencies = [
+    "fastapi[standard]>=0.115.12",
+    "httpx>=0.28.1",
+    "pytest>=8.3.5",
```

Comment:
pytest should be a dev dependency, not included in the main project deps. Remove it and then run `uv add --dev pytest`.